js/_actions.js

```diff
@@ -0,0 +1,142 @@
+// polyfills
+require('babel-polyfill');
+var fetch = require('isomorphic-fetch');
+
+// checks if a response from fetch worked
+function checkStatus(response) {
+	if (response.status >= 200 && response.status < 300) {
+		return response;
+	}
+	var error = new Error(response.statusText);
+	error.response = response;
+	throw error;
+}
+
+export const SET_MESSAGE = "SET_MESSAGE";
+export function setError(errorMessage) {
+	return {
+		type: SET_MESSAGE,
+		message: errorMessage,
+		message_type: "error"
+	};
+}
+
+export const SET_API_ENDPOINT = "SET_API_ENDPOINT";
+export function setAPIEndpoint(url) {
+	return {type: SET_API_ENDPOINT, url: url};
+}
+
+export const SET_ACTIVE_STEP = "SET_ACTIVE_STEP";
+export function setActiveStep(id) {
+	return {type: SET_ACTIVE_STEP, id};
+}
+
+export const GIT_UPDATE_REQUEST = 'GIT_UPDATE_REQUEST';
```

Comment:
Me and sean prefer the verb first (imperative), followed by what the action is about.
=> START_GIT_UPDATE